docs/examples/move/nft_marketplace/README.md

```diff
@@ -0,0 +1,179 @@
+# Marketplace Guide
+
+The `marketplace_extension.move` module provides a straightforward implementation of a marketplace extension. To utilize it, follow the steps outlined below.
+The `item_for_market.move` contains mocked item data for use within the marketplace.
+The `rental_extension.move` is an extension adds functionality to enable item rentals.
+
+## Steps to Use the Marketplace
+
+### 1. Connect to the Network
+
+Connect to the IOTA network (e.g., using a faucet to obtain tokens).
+
+### 2. Install Kiosk
+
+By installation, we mean creating a Kiosk object and an OwnerCap, then transferring them to the caller.
+Run the following command to install the Kiosk module:
+
+```bash
+iota client call \
+    --package 0x2 \
+    --module kiosk \
+    --function default
+```
+
+### 3. Publish `nft_marketplace` package
+
+#### 3.1(Optional) Publish Kiosk rules modules if these are not present in the network you are using
+
+Publish Kiosk rules modules(package):
+
+```bash
+iota client publish iota/kiosk
+```
+
+After publishing, export the following variable:
+
+- `RULES_PACKAGE_ID`: The ID of rules package.
```

Comment:
Let's add a command here, for example:
```sh
export RULES_PACKAGE_ID =<PACKAGE_ID>
```
We also can add a short command output like [this](https://github.com/iotaledger/iota/blob/d24f36b693efc1627e56b4cddbb7e59104d4a797/docs/content/developer/getting-started/publish.mdx#L24) to clarify where the ID can be taken:
```
╭─────────────────────────────────────────────────────────────────────╮
│ Object Changes                                                      │
├─────────────────────────────────────────────────────────────────────┤
│ Created Objects:                                                    │
│  ...                                                                │
|                                                                     |
│ Mutated Objects:                                                    │
│  ...                                                                │
|                                                                     |
│ Published Objects:                                                  │
│  ┌──                                                                │
│  │ PackageID: <PACKAGE_ID>                                          │
│  │ Version: 1                                                       │
│  │ Digest: <DIGEST_HASH>                                            │
│  │ Modules: first_package                                           │
│  └──                                                                │
╰─────────────────────────────────────────────────────────────────────╯
```
I see we have several export steps in the document, so it doesn't make sense to add this for all of them. This comment is for your consideration.